generators/add-extension/index.js

```diff
@@ -110,10 +110,10 @@ class Generator extends Base {
   }
 
   _injectWebExtension(d) {
-    const file = this.destinationPath('phovea.js');
+    const file = this.destinationPath('src/phovea.ts');
     const old = this.fs.read(file);
     const absFile = d.module.startsWith('~') ? d.module.slice(1) : `./src/${d.module.includes('.') ? d.module.slice(0, d.module.lastIndexOf('.')) : d.module}`;
-    const text = `\n\n  registry.push('${d.type}', '${d.id}', function() { return import('${absFile}'); }, ${d.stringify(d.extras, ' ')});\n  // generator-phovea:end`;
+    const text = `\n\n  registry.push('${d.type}', '${d.id}', function() { return System.import('${absFile}'); }, ${d.stringify(d.extras, ' ')});\n  // generator-phovea:end`;
```

Comment:
We might not need to add the `System` as it will be deprecated with Webpack 4. See https://github.com/phovea/generator-phovea/issues/286#issuecomment-566553497